src/main/java/org/wso2/carbon/inbound/salesforce/poll/SalesforceStreamData.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why you are changing the license year keep it as 2015

Reply:
fixed